### ISSUE TITLE
Bug Report: Pieces reported as stopped when they're not

### ISSUE THREAD

**Issue by PeterC89** [nrkno/sofie-core#1153]
### About me

This bug report is posted on behalf of the BBC

### Observed Behavior

Pieces that are stopped either via `stopPiecesOnLayers` or `stopPieceInstances` don't always stop.
They continue to show in the GUI as playing and the timeline objects continue to exist but the database shows them as stopped and thus they cannot be stopped without inserting another piece on the same layer.
```json
{
    "_id": "_OPGI0kSC8f98MBOHDY3zSJOXHc__s7G93nXmWcvFivTbe_jv33kKaXSuNunW5AG",
    "rundownId": "2kgghfTRYjAlP5g9OHExeWfRfz0_",
    "playlistActivationId": "u6hBjd8Gxcbb3tK6F",
    "partInstanceId": "_OPGI0kSC8f98MBOHDY3zSJOXHc__s7G93nXmWcvFivTbe",
    "piece": {
        "pieceType": "normal",
        "externalId": "4QKGx03_lgcT8d2RPgoWwncw7Ec__audio",
        "enable": {
            "start": 39290
        },
        "name": "OS 2 Audio",
        "sourceLayerId": "audio_os_2",
        "outputLayerId": "pgm_audio",
        "content": {},
        "lifespan": "part-only",
        "_id": "jv33kKaXSuNunW5AG",
        "startPartId": "_OPGI0kSC8f98MBOHDY3zSJOXHc_",
        "invalid": false
    },
    "dynamicallyInserted": 1707217468628,
    "reportedStartedPlayback": 1707217468743.5,
    "plannedStartedPlayback": 1707217468743.5,
    "reportedStoppedPlayback": 1707217468818.5,
    "plannedStoppedPlayback": 1707217468818.5
}
```

For example stopping 5 layers using `stopPiecesOnLayers` results in core reporting the cropping of only 4 layers
```json
{"level":"info","message":"Blueprint action: Cropping PieceInstance \"_OPGI0kSC8f98MBOHDY3zSJOXHc__sYbadkxf8fXn4bLg8_CypP6SRsfsTQeRfnq\" to 1707223033540","source":"studio:studio0"}
{"level":"info","message":"Blueprint action: Cropping PieceInstance \"_OPGI0kSC8f98MBOHDY3zSJOXHc__sYbadkxf8fXn4bLg8_jtRBkGh2DCFDe2STu\" to 1707223033540","source":"studio:studio0"}
{"level":"info","message":"Blueprint action: Cropping PieceInstance \"_OPGI0kSC8f98MBOHDY3zSJOXHc__sYbadkxf8fXn4bLg8_8tSjLkceoeaDq8ZRT\" to 1707223033540","source":"studio:studio0"}
{"level":"info","message":"Blueprint action: Cropping PieceInstance \"_OPGI0kSC8f98MBOHDY3zSJOXHc__sYbadkxf8fXn4bLg8_6xoYTX85PWTMqXpGo\" to 1707223033540","source":"studio:studio0"}
```

### Expected Behavior

Pieces should stop when requested

### Version

release51

### Severity / Impact

Not currently an issue as enabling multi-gateway mode 'fixes' this issue
Multi-gateway mode also appears to fix #1118

### Level of Involvement

Just raising awareness

**Comment by Julusian** [comment]
@PeterC89 This sounds like this could be the same issue as we have seen at NRK. The fix for that was 7485d2b78db95e15c9fb2a77d3015053ba35d907, could you retest with that fix to see if that fix resolves this?

**Comment by PeterC89** [comment]
I've just tested and am still seeing this problem sadly even with multi-gateway mode enabled 😞
